public/components/FormControls/FormikCheckbox/FormikCheckbox.test.js

```diff
@@ -0,0 +1,32 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import React from 'react';
+import { render } from 'enzyme';
+import { Formik } from 'formik';
+
+import FormikCheckbox from './FormikCheckbox';
+
+describe.skip('FormikCheckbox', () => {
```

Comment:
Why the test is being skipped ?

Reply:
Also, could you add test cases to cover the branches which match snapshot.

Reply:
Sorry, forgot to add it. Will change.